support/chez/support.ss

```diff
@@ -460,38 +460,34 @@
   '()))
 
 (define (blodwen-channel-get-with-timeout ty chan timeout)
-  (let loop ()
-    (let* ([sec (div timeout 1000)])
+  ;; timeout is in milliseconds, convert to nanoseconds
+  (let* ([timeout-ns (* timeout 1000000)]
+         [sleep-ns 10000] ; 10 us step
+         [sleep-time (make-time 'time-duration (mod sleep-ns 1000000000)
+                                                (div sleep-ns 1000000000))])
```

Comment:
Given that `sleep-ns` is a constant, aren't these values going to be exactly `10000` & `0`?

Reply:
In #3577 you mentioned an "exponential backoff" but I guess it did not make it in the PR?

Reply:
> Given that `sleep-ns` is a constant, aren't these values going to be exactly `10000` & `0`?

Yeah, `sleep-ns` is a constant 10 us sleep, so yes I believe so